main.js

```diff
@@ -0,0 +1,88 @@
+console.log(`Sum = `, +(0.1 + 0.2).toFixed(2));
+
+const yourLogin = prompt(`Ведіть Ваш логін`);
+const yourPassword = prompt(`Ведіть Ваш пароль`);
+
+const login = "admin";
+const password = "12pass33210";
+
+if (yourLogin !== login && yourPassword !== password) {
+  console.log(`Нажаль, ви не маєте доступу до адмін-панелі.`);
+} else {
+  yourLogin === login && yourPassword === password
+    ? console.log(`Ви успішно увійшли`)
+    : console.log(`Проблема з логіном або паролем`);
+}
+
+const money = +prompt(`Скільки в тебе грошей ?`);
+const watermelonPrice = 21;
+const carrotPrice = 8;
+const potatoesPrice = 12;
+
+const watermelon = "кавун";
+const carrot = "морква";
+const patatoes = "картопля";
+
+console.log(
+  `Кавун = ${watermelonPrice} грн`,
+  `Морква = ${carrotPrice} грн`,
+  `Картопля = ${potatoesPrice} грн`
+);
+
+if (money < watermelonPrice && money < carrotPrice && money < potatoesPrice) {
```

Comment:
No need to check every product price

```suggestion
if (money < carrotPrice) {
...
}
```